homeassistant/components/asterisk_mbox/__init__.py

```diff
@@ -39,9 +40,9 @@
 )
 
 
-def setup(hass, config):
+def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up for the Asterisk Voicemail box."""
-    conf = config.get(DOMAIN)
+    conf = config[DOMAIN]
```

Comment:
Can you guarantee that the DOMAIN key is set?  Not sure why the `.get()` was removed here.

Reply:
If `DOMAIN` wasn't in the config, it shouldn't/wouldn't have run this setup method.
Even if it did, it would then have failed on the next line: `host = conf[CONF_HOST]`